Mathlib/NumberTheory/SelbergSieve.lean

```diff
@@ -89,6 +89,21 @@ class SelbergSieve extends BoundingSieve where
 
 attribute [arith_mult] BoundingSieve.nu_mult
 
+namespace Mathlib.Meta.Positivity
+
+open Lean Meta Qq
+
+/-- Extension for the `positivity` tactic: `BoundingSieve.weights`. -/
+@[positivity BoundingSieve.weights _]
+def evalBoundingSieveWeights : PositivityExt where eval {u α} _zα _pα e := do
```

Comment:
@FLDutchmann This positivity extension looks good to me - would you like to give this another look? Two cargo-cult reviews is certainly good enough to maintainer merge.